tests/README.md

```diff
@@ -21,7 +21,8 @@ tests
 
 ## Running the maven Tests without cucumber tags
 
-Add values to the keys in config.properties file
+1. Add values to the keys in config.properties file
```

Comment:
The markup suggestion here from code climate. Not sure if you've seen or dealt with the warnings it places inline on the diff yet. Click on the yellow bar to expand the message.

Reply:
done. Thanks.